### ISSUE TITLE
Suggestion: Onboarding should ask about light/dark/high contrast themes

### ISSUE THREAD

**Issue by solarbirdy** [mastodon/mastodon#27278]
### Pitch

While many (most?) existing users of the Mastodon Dark theme prefer it - including myself - a substantial number of people have vision issues which make dark theme difficult to use. Some other users simply prefer the light theme, but this feature request is mostly about accessibility.

Computer users in general have a good understanding at this point about "light mode," "dark mode," and "accessibility" modes. It's a broad enough understanding that Apple asks its users this question during new hardware onboarding procedures with good success. They also ask it during some OS upgrades on desktop.

I suggest that Mastodon new-account onboarding - particularly but not just on the web UI - include this as an onboarding step, to help those with vision issues which are made worse by "dark mode" themes. The mode could include a preview of a sample timeline in each mode, with a simple one-click selector.

### Motivation

I have personally heard from people with visual processing issues - not necessarily limited vision, but issues like dyslexia - for whom the default "Dark" theme is a meaningful problem.

(One was just today, which is what sent me here. She's solved her issues on her own - and it was more than just a "light mode" / "dark mode" issue - but the default of "dark mode" was a problem for her. The new search functionality dropdown is a _particular_ issue.)

I think a "light mode/dark mode/high contrast mode" onboarding question would help address this for a small but important percentage of possible new users, and would do so in a way that most users already find familiar.

As a side benefit, outside of the tech community, a lot of people really do prefer 'light mode' and I think having the ability to turn that on would be an option many new users would appreciate.

**Comment by FediVideos** [comment]
Agreed, had someone asking on Feditips recently who had problems with dark mode and their eyesight. They didn't know about light mode, but as soon as they switched to light mode it solved their eyesight issues.
